docs/installation/prepare/jetson-agx-orin-setup.md

```diff
@@ -109,6 +109,14 @@ For a Jetson AGX Orin, reference the following:
 Note that I2C buses do not need to be configured through <file>jetson-io.py</file>.
 See NVIDIA's documentation on [Configuring the Jetson Expansion Headers](https://docs.nvidia.com/jetson/archives/r35.1/DeveloperGuide/text/HR/ConfiguringTheJetsonExpansionHeaders.html) for more information.
 
+## Next Steps
+
+{{< cards >}}
+{{% card link="/manage/configuration/" %}}
+{{% card link="/tutorials/" %}}
+{{% card link="/try-viam/" %}}
```

Comment:
With `viam-server` installed on my own hardware, much of the allure and immediate usefulness of Try is lost. How about Program (SDKs) instead?

```suggestion
{{% card link="/program/" %}}
```

recommend making this change on all the other prepare board procedurals too.